From 3065554afa5f3a74035e6735dda06ec68ce2e38c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Luk=C3=A1=C5=A1=20Tinkl?= <lukas@kde.org>
Date: Tue, 29 Apr 2014 00:23:29 +0200
Subject: [PATCH 04/10] connect to the right signal

when updating the _device_ properties (ie. the remaining time),
only happens with UPower >= 0.99
---
 powerdevil/daemon/backends/upower/powerdevilupowerbackend.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/powerdevil/daemon/backends/upower/powerdevilupowerbackend.cpp b/powerdevil/daemon/backends/upower/powerdevilupowerbackend.cpp
index b592254..fd402c6 100644
--- a/powerdevil/daemon/backends/upower/powerdevilupowerbackend.cpp
+++ b/powerdevil/daemon/backends/upower/powerdevilupowerbackend.cpp
@@ -465,7 +465,7 @@ void PowerDevilUPowerBackend::slotDeviceAdded(const QString & device)
 
     // for UPower >= 0.99.0 which doesn't emit the DeviceChanged(QString) signal
     QDBusConnection::systemBus().connect(UPOWER_SERVICE, device, "org.freedesktop.DBus.Properties", "PropertiesChanged", this,
-                                         SLOT(onPropertiesChanged(QString,QVariantMap,QStringList)));
+                                         SLOT(onDevicePropertiesChanged(QString,QVariantMap,QStringList)));
 
     updateDeviceProps();
 }
-- 
1.9.0

